core/src/main/scala/kafka/tools/ConsoleProducer.scala

```diff
@@ -178,8 +178,8 @@ object ConsoleProducer {
     val requestRequiredAcksOpt = parser.accepts("request-required-acks", "The required acks of the producer requests")
       .withRequiredArg
       .describedAs("request required acks")
-      .ofType(classOf[java.lang.Integer])
-      .defaultsTo(0)
+      .ofType(classOf[java.lang.String])
+      .defaultsTo("0")
```

Comment:
We no longer need to call `toString` in the following `config.requestRequiredAcks.toString`.